src/seed-users.ts

```diff
@@ -0,0 +1,41 @@
+import * as dotenv from 'dotenv';
+import { faker } from '@faker-js/faker';
+import { setupDBConnection } from './data-source';
+import { User } from './User';
+import { passwordHashing, userRepository } from './utils';
+
+dotenv.config();
+
+function randomUser(): User {
+  const user = new User();
+
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+
+  user.name = firstName + ' ' + lastName;
+  user.email = faker.internet.email(faker.internet.userName(firstName, lastName));
+  user.password = passwordHashing(faker.internet.password());
+  user.birthDate = faker.date.birthdate({ min: 18, max: 65, mode: 'age' }).toLocaleDateString('pt-BR');
+
+  return user;
+}
+
+async function populateDb(userPopulation: number) {
+  const userArray = [];
```

Comment:
```suggestion
  const users = [];
```
uma boa prática para nomear uma variável é não acoplar o "tipo" ao nome, mas sim o propósito, dai um array pode ser perfeitamente entendido com o nome no plural, por exemplo. Ou então um [hashmap](https://www.tutorialspoint.com/how-to-use-hashmap-in-typescript) poderia ser nomeado como `<resource>By<key>` ao invés de `<resource>hashMap`, entre outros

Reply:
Matheus, valeu por me passar essa convenção!